sklearn/utils/arpack.py

```diff
@@ -1104,18 +1099,21 @@ def _eigs(A, k=6, M=None, sigma=None, which='LM', v0=None, ncv=None,
         For a real matrix A, shift-invert can either be done in imaginary
         mode or real mode, specified by the parameter OPpart ('r' or 'i').
         Note that when sigma is specified, the keyword 'which' (below)
-        refers to the shifted eigenvalues w'[i] where:
+        refers to the shifted eigenvalues w'[i] where
```

Comment:
This had made the script sad... But now that we have removed the script for now, this is back in!